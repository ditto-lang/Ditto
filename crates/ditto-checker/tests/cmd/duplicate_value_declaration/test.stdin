module Test exports (..) a : Bool = true a : Int = 5
