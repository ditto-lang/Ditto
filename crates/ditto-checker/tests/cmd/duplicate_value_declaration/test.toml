bin.name = "ditto-checker-testbin-check-module"
args = ["duplicate_value_declaration.ditto"]
fs.sandbox = true
status = "failed"
