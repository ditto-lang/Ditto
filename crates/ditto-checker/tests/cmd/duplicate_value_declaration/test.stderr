
  × duplicate top-level name
   ╭─[duplicate_value_declaration.ditto:1:1]
 1 │ module Test exports (..); a : Bool = true; a : Int = 5;
   ·                           ┬                ┬
   ·                           │                ╰── can't be redefined here
   ·                           ╰── previously defined here
   ╰────

